mobile/pubspec.yaml

```diff
@@ -47,6 +47,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  pedantic: ^1.9.0
```

Comment:
since we're importing `package:pedantic/pedantic.dart`, we should move this as a normal dependency: https://github.com/google/pedantic#using-the-lints